cnames_active.js

```diff
@@ -294,6 +294,7 @@ var cnames_active = {
   "awoo": "awoojs.github.io/website",
   "aws-auth": "iamarkadyt.github.io/aws-auth",
   "axii": "ariesate.github.io/axii-site",
+  "axios-cache-lite": "nom-nom-hub.github.io/axios-cache-lite",
   "axios-cache-interceptor": "arthurfiorette.github.io/axios-cache-interceptor",
```

Comment:
```suggestion
  "axios-cache-interceptor": "arthurfiorette.github.io/axios-cache-interceptor",
  "axios-cache-lite": "nom-nom-hub.github.io/axios-cache-lite",
```
Ah yes, these need to be swapped to be in alphabetical order

Reply:
Sorry about this im new to this im a great coder but I've never really got into this until recently and im still learning to do things on GitHub the right way. if I need to personally do anything myself please do let me know. im winging it right now lol. I appreciate your help and support!